tests/testthat/test_Spectrum_Constructor.R

```diff
@@ -126,6 +128,30 @@ test_that("C-level multi-Spectrum1 constructor with M/Z ordering", {
         expect_identical(mz(spectL[[i]]), mzValsList[[i]][idxList[[i]]])
         expect_identical(intensity(spectL[[i]]), intValsList[[i]][idxList[[i]]])
     }
+
+    ## Check empty spectra.
+    ## Spectrum1
+    mzVals <- sort(abs(rnorm(200, mean = 100, sd = 10)))
```

Comment:
Should we use `set.seed` here? Otherwise if a test randomly fails we would never able to reproduce it because we don't know the input values.